### PR TITLE
Zkverify sindri example

### PR DESCRIPTION
**Pull request by rushby**

### CODE REVIEW

File: compile.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import sindri from 'sindri';
+import 'dotenv/config';
+
+// Create a Sindri Client using your API KEY.
+const myClient = sindri.create({ apiKey: process.env.SINDRI_API_KEY}, undefined);
```
Comment (yurikoinaba) [inline compile.ts:5]: `undefined` doesn't need to be passed in.

Comment (yurikoinaba) [reply compile.ts:5]: Could also be good to link to the official docs for additional reference 
https://sindri.app/docs/reference/sdk/ts/classes/SindriClient/#create

and

https://sindri.app/docs/reference/sdk/ts/classes/SindriClient/#createcircuit

File: prove.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+async function generateProof(X: number = 23, Y: number = 52): Promise<any> {
+    const circuitIdentifier = 'is-equal:latest';
+    const proofInput = JSON.stringify({ X, Y });
+
+    // Create a Sindri Client using your API KEY.
+    const myClient = sindri.create({ apiKey: process.env.SINDRI_API_KEY }, undefined);
+
+    try {
+        // Create the proof using Sindri
+        const proof = await myClient.proveCircuit(circuitIdentifier, proofInput);
```
Comment (yurikoinaba) [inline prove.ts:15]: Same here, `undefined` isn't needed.

https://sindri.app/docs/reference/sdk/ts/classes/SindriClient/#provecircuit

File: README.md [hunk 3]
```diff
@@ -1 +1,44 @@
...
+1. Install Dependencies
+```shell
+npm install
+```
+
+2. Environment Variables
+- Set `SINDRI_API_KEY` environment variable (e.g. in .env file) 
+- Set zkVerify wallet `SEED_PHRASE` environment variable (e.g. in .env file)
+
+3. Compile & Deploy the circuit on Sindri
```
Comment (yurikoinaba) [inline README.md:14]: Add a note that this example uses groth16

File: pages/api/sindri.ts [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        return res.status(500).json({ error: error.message || 'An unknown error occurred.' });
+    }
+}
+
+async function pollForProof(proofId: string) {
+    const proofDetailUrl = `https://sindri.app/api/v1/proof/${proofId}/detail`;
+    const MAX_POLLING_ATTEMPTS = 10;
+    const POLLING_INTERVAL = 5000;
+
+    for (let attempt = 0; attempt < MAX_POLLING_ATTEMPTS; attempt++) {
```
Comment (yurikoinaba) [inline pages/api/sindri.ts:50]: Maybe add a comment that the proof generation is expected to take a while

**Review by yurikoinaba — APPROVED** [review]

File: compile.ts (hunk 1 above)
Comment (rushby) [reply compile.ts:5]: Will do and strange my IDE complained about that value being missing! 😄

**Pull request merged**